media/views/oss/suggestion/suggestion.css

```diff
@@ -0,0 +1,141 @@
+html {
+  height: 100%;
+  font-size: 62.5%;
+}
+body {
+  height: 100%;
+  padding: 0;
+  font-size: 1.4rem;
+  line-height: 1.5;
+}
+body * {
+  box-sizing: border-box;
+}
+
+h2 {
+  font-size: 1em;
+  margin-block-start: 0;
+}
+
+a {
+  cursor: pointer;
+  color: #7c74f2;
```

Comment:
would be nice to use a `const` for HEX colors once we switch to SASS ;)

Reply:
oh definitely! The ticket is in the backlog :)